3rdlib/quazip/quazip.pro

```diff
@@ -10,6 +10,8 @@ CONFIG += staticlib
 
 INCLUDEPATH += ../zlib
 
+QMAKE_CXXFLAGS += -std=c++11
+
```

Comment:
MSVC just simply ignored the `-std=c++11` with a warning, so basically it's fine.